### PR TITLE
Convert partners to new schema, Fixes #328

### LINKED ISSUE #328: wi/partner should pull partners for the current event.
Currently the partner listing page just pulls _all_ partners. We need it pull _this_ year's partners.

**Query**
```
  query getPartners($eventId: ID!) {
    events {
      event(id: $eventId) {
        get {
        id
        name
        partners {
          [required fields]
        }
       }
      }
    }
  }
```
**Variables**
```
{
	"eventId": "ByE7Dc7eCGcRFzLhWhuI"
}
```

### PR DESCRIPTION
**Pull request by GeekOnCoffee**

### CODE REVIEW

**Comment by csell5** [comment]
#### 📝Changed routes:


| [**/wi/partners**](https://that-website-git-fork-geekoncoffee-feature-limit-part-055aa8.that-conference.now.sh/wi/partners) |
|:-:|
|<a href="https://that-website-git-fork-geekoncoffee-feature-limit-part-055aa8.that-conference.now.sh/wi/partners">Screenshot of /wi/partners</a><br /><sup><a href="https://images.weserv.nl?url=https://microlink.nyc3.digitaloceanspaces.com/vHP014UnTo8iW3Q1Icn9ghABqPM_">(view full size)</a> |







Commit 99c03a3dfe1bc9d4d292085a1c9d14c3f1a46654 (https://that-website-git-fork-geekoncoffee-feature-limit-part-055aa8.that-conference.now.sh).

File: pages/wi/partners.js [hunk 1]
```diff
@@ -5,23 +5,33 @@ import { useQuery } from '@apollo/react-hooks';
...
-        partnershipLevel
-        companyName
-        companyLogo
-        heroImage
-        website
+  query getPartners($eventId: ID!) {
+    events {
+      event(id: $eventId) {
+        get {
+          id
```
Comment (brettski) [inline pages/wi/partners.js]: Add year of the event so it may be referenced elsewhere in the page

File: pages/wi/partners.js [hunk 2]
```diff
@@ -99,18 +109,29 @@ const renderPartner = (
...
 
-  const partners = data.partners.all;
+  dlog('partners %o', partners);
   return (
     <div>
       <ContentSection>
         <Grid columns="repeat(auto-fit,minmax(32rem,1fr))">
           <Cell>
-            <Header>2019 Partners</Header>
+            <Header>2020 Partners</Header>
```
Comment (brettski) [inline pages/wi/partners.js]: Use Queried `year` instead of hard-coding it.

**Comment by GeekOnCoffee** [comment]
Ideally get #337 done for this

**Pull request merged**